test/functional/test_framework/mininode.py

```diff
@@ -654,3 +655,11 @@ def on_inv(self, message):
     def get_invs(self):
         with mininode_lock:
             return list(self.tx_invs_received.keys())
+
+    def wait_for_broadcast(self, txns):
+        # Waits for the txns (txids) to complete initial broadcast.
+        # Wait until invs have been received (and getdatas sent) for each txid.
+        self.wait_until(lambda: set(self.get_invs()) == set([int(tx, 16) for tx in txns]), 60)
```

Comment:
```suggestion
        self.wait_until(lambda: set(self.get_invs()) == set([int(tx, 16) for tx in txns]), timeout)
```

I think it makes sense to define a default value of `timeout=60` for the wait_for_broadcast member function. This way, callers can decide to overwrite it if needed.